### DISCUSSION TITLE
Composite primary keys in Eloquent

### DISCUSSION THREAD

**Discussion by pingumen96** [laravel/framework#45897] [Polls]
Hello everyone, I'm a Laravel user from version 5.
It made a lot of progress but I still find incredible that composite primary keys are not supported by default, because I still have to hack my way around for what I consider a common practice.
So i decided to create this poll, just to understand what does people think about it.

**Comment by henzeb** [comment]
Not by default as default implies standard instead of a single key.

But yes, it should support composite keys.

**Reply by pingumen96** [reply]
> Not by default as default implies standard instead of a single key.

You're right, obviously I wasn't meaning that.

**Comment by groovenectar** [comment]
What is the best argument that it should not?

**Comment by macropay-solutions** [comment]
https://github.com/laravel/framework/discussions/48152#discussioncomment-8064510
